examples/flutter_gallery/lib/demo/playground/configuration/material_helpers.dart

```diff
@@ -0,0 +1,297 @@
+  IconChoice({@required this.icon, @required this.code});
+  IconData icon;
+  String code;
+}
+
+final List<BorderChoice> borderChoices = <BorderChoice>[
+  BorderChoice(type: 'square', code: '''
+RoundedRectangleBorder(
+  borderRadius: BorderRadius.zero
+)'''),
+  BorderChoice(type: 'rounded', code: '''
+RoundedRectangleBorder(
+  borderRadius: BorderRadius.circular(10.0)'
+)'''),
+  BorderChoice(type: 'beveled', code: '''
+BeveledRectangleBorder(
+  borderRadius: BorderRadius.circular(10.0)
+)'''),
+  BorderChoice(type: 'stadium', code: '''
+StadiumBorder()'''),
+];
+
+final List<ColorChoice> colorChoices = <ColorChoice>[
+  ColorChoice(color: Colors.white, code: 'Colors.white'),
+  ColorChoice(color: Colors.orange, code: 'Colors.orange'),
+  ColorChoice(color: Colors.cyan[200], code: 'Colors.cyan[200]'),
+  ColorChoice(color: Colors.lightBlue[300], code: 'Colors.lightBlue[300]'),
+  ColorChoice(color: Colors.blue, code: 'Colors.blue'),
+  ColorChoice(color: Colors.blue[800], code: 'Colors.blue[800]'),
+];
+
+final List<IconChoice> iconChoices = <IconChoice>[
+  IconChoice(icon: Icons.thumb_up, code: 'Icons.thumb_up'),
+  IconChoice(icon: Icons.android, code: 'Icons.android'),
+  IconChoice(icon: Icons.alarm, code: 'Icons.alarm'),
+  IconChoice(icon: Icons.accessibility, code: 'Icons.accessibility'),
+  IconChoice(icon: Icons.call, code: 'Icons.call'),
+  IconChoice(icon: Icons.camera, code: 'Icons.camera'),
+];
+
+final List<String> borderOptions =
+    borderChoices.map((BorderChoice b) => b.type).toList();
+
+final List<Color> colorOptions =
+    colorChoices.map((ColorChoice c) => c.color).toList();
+
+final List<IconData> iconOptions =
+    iconChoices.map((IconChoice i) => i.icon).toList();
+
+String codeSnippetForColor(Color color) {
+  return colorChoices
+      .where((ColorChoice c) => c.color == color)
+      .toList()[0]
+      .code;
+}
+
+String codeSnippetForBorder(String type) {
+  return borderChoices
+      .where((BorderChoice b) => b.type == type)
+      .toList()[0]
+      .code;
+}
+
+String codeSnippetForIcon(IconData icon) {
+  return iconChoices.where((IconChoice b) => b.icon == icon).toList()[0].code;
+}
+
+Widget sliderPicker({
+  @required String label,
+  double value = 0.0,
+  double minValue = 0.0,
+  double maxValue = 1.0,
+  int divisions,
+  ValueChanged<double> onValueChanged,
+}) {
+  return PropertyColumn(
+    label: label,
+    widget: Padding(
+      padding: const EdgeInsets.only(bottom: 5.0),
+      child: Slider(
+        value: value,
+        min: minValue,
+        max: maxValue,
+        divisions: divisions,
+        onChanged: onValueChanged,
+      ),
+    ),
+  );
+}
+
+ShapeBorder borderShapeFromString(String type, [bool side = true]) {
+  ShapeBorder shape;
+
+  BorderSide borderSide = const BorderSide(
+    color: Colors.grey,
+    width: 2.0,
+  );
+
+  borderSide = side ? borderSide : BorderSide.none;
+
+  switch (type) {
+    case 'square':
+      shape = RoundedRectangleBorder(
```

Comment:
just `return RoundedRectangleBorder(borderRadius: BorderRadius.zero, side: borderSide);`

Likewise for the rest of switch clauses

Reply:
Updated and works, but linter says: 

```[dart] This function has a return type of 'ShapeBorder', but doesn't end with a return statement. [missing_return]```